lang/en/kanban.php

```diff
@@ -123,6 +123,14 @@
 $string['messageprovider:due'] = 'Card due';
 $string['messageprovider:moved'] = 'Card moved';
 $string['modulename'] = 'Kanban board';
+$string['modulename_help'] = 'This activity supports using kanban method for managing projects or learning processes.
```

Comment:
This is not properly being formatted in the info box, looks like the markdown list is not supported. Could this be solved differently?

Reply:
This actually belongs to #62. Seems to have somehow got into the commit here?

Reply:
I was already wondering :)
Can you fix the history please? And fix formatting in #62 :)